### PR TITLE
Feature12

### PR DESCRIPTION
**Pull request by muralidhuli**
feature12 updated 
files changed for practice

### CODE REVIEW

File: feature1.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Hello feature1 feature12
```
Comment (techsolutionsapp2) [inline feature1.txt]: Please change feature12 to 1-2

File: feature1.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+featur 12 updated.
```
Comment (techsolutionsapp2) [inline feature1.txt:3]: update 1-2-3

**Review by techsolutionsapp2 — CHANGES_REQUESTED** [review]
please do the necessary changes

File: feature1.txt (hunk 1 above)
Comment (muralidhuli) [reply feature1.txt]: review comment fixed

File: feature1.txt (hunk 2 above)
Comment (muralidhuli) [reply feature1.txt:3]: as per standards

**Review by techsolutionsapp2 — APPROVED** [review]

**Pull request merged**